### PR TITLE
positioning of light signup after third paragraph or 50% whichever is lower

### PR DESCRIPTION
**Pull request by apaleslimghost**
currently makes no attempt to avoid placing next to inline related article or other asset

### CODE REVIEW

**Comment by georgecrawford** [comment]
@quarterto Build failed on Heroku. Also - do you want to give this to QA to test 10 or so articles for sanity?

**Comment by apaleslimghost** [comment]
> Unknown plugin "transform-object-rest-spread"
> 
> `"babel-plugin-transform-object-rest-spread": "^6.8.0",`

hmm

**Comment by apaleslimghost** [comment]
Btw I'm still working on not positioning it next to other assets, but I'm running into Cheerio problems: cheeriojs/cheerio#890

File: package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
     "@quarterto/promise-loop-interval": "^1.0.0",
     "babel-cli": "^6.6.5",
     "babel-plugin-transform-runtime": "^6.7.5",
+    "babel-plugin-transform-object-rest-spread": "^6.8.0",
```
Comment (apaleslimghost) [inline package.json:32]: @georgecrawford doh

**Pull request merged**